doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
-via their respective CNI integration.
 
-> Note: An SDN solution can manage the pod networks via the Kubernetes API or
-can have a CNI integration.
+| Requirement | Networking Solution with Multus | Networking Solution with DANM |
+|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported |
+| Security Groups (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> |
+| SFC support (`infra.net.cfg.005`) | Not relevant | Not relevant |
+| Traffic patterns symmetry (`infra.net.cfg.006`) | Not relevant | Not relevant |
+| Network resiliency (`req.inf.ntw.01`) | Supported | Supported |
+| Centrally administrated and configured (`req.inf.ntw.03`) | Supported via Kubernetes API | Supported via Kubernetes API |
+| Dual stack IPv4 and IPv6 for Kubernetes workloads (`req.inf.ntw.04`) | Supported via the additional CNI plugin | Supported |
+| Integrating SDN controllers (`req.inf.ntw.05`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| More than one networking solution (`req.inf.ntw.06`) | Supported | Supported |
+| Choose whether or not to deploy more than one networking solution (`req.inf.ntw.07`) | Supported | Supported |
+| Kubernetes network model (`req.inf.ntw.08`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| Do not interfere with or cause interference to any interface or network it does not own (`req.inf.ntw.09`) | Supported | Supported |
+| Cluster wide coordination of IP address assignment (`req.inf.ntw.10`) | Supported via the additional CNI plugin | Supported |
 
-The container based architecture must support telecom equipment networking where
-the CNF networks are set up by the operator's network administrators. This is
-why, as `req.gen.cnt.05` requires, the architecture must provide a set of
-abstract management APIs to manage the network connectivity of the CNF pods
-themselves.
+<p align="center"><b>Table 3-1:</b> Comparison of CNI multiplexers/meta-plugins within the networking solution</p>
 
-The API must support multiple tenants and must require elevated access rights to
-manipulate infrastructure related API objects as these operations generally
-require reconfiguration of the physical network infrastructure.
+(1): Under implementation in the current release.  
 
-To fulfill the requirements of `e.cap.016` the architecture should optionally
-support the use of device plugins via the Device Plugin API and the alignment of
-the devices, CPU topology and Huge Pages must be supported using the [Topology
-Manager](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/).
-
-The architecture must support IPv4, IPv6 and dual stack interfaces of the
```

Comment:
I think this should not be removed.

Reply:
Another mistake, sorry - have added to chapter 4 in https://github.com/cntt-n/CNTT/commit/719956b66256e0e10c06b64e5097c5a0dbab22bf

Reply:
Is this ok to resolve @CsatariGergely?